inference-engine/src/transformations/src/transformations/op_conversions/mvn_decomposition.cpp

```diff
@@ -0,0 +1,68 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "transformations/op_conversions/mvn_decomposition.hpp"
+
+#include <memory>
+
+#include <ngraph/opsets/opset6.hpp>
+#include <ngraph/rt_info.hpp>
+#include <ngraph/pattern/op/wrap_type.hpp>
+
+NGRAPH_RTTI_DEFINITION(ngraph::pass::MVNDecomposition, "MVNDecomposition", 0);
+
+ngraph::pass::MVNDecomposition::MVNDecomposition() {
+    // Decomposes MVN(x, axes) op into sub-graph (x - ReduceMean(x, axes)) / Sqrt(ReduceSum((x - ReduceMean(x, axes)) ^ 2))
+    auto mvn = ngraph::pattern::wrap_type<opset6::MVN>();
+
+    ngraph::matcher_pass_callback callback = [=](ngraph::pattern::Matcher& m) {
+        auto& pattern_to_output = m.get_pattern_value_map();
+        auto mvn_node = std::dynamic_pointer_cast<ngraph::opset6::MVN>(pattern_to_output.at(mvn).get_node_shared_ptr());
+
+        if (mvn_node == nullptr || m_transformation_callback(mvn)) {
+            return false;
+        }
+
+        auto data = mvn_node->input_value(0);
+        auto axes = mvn_node->input_value(1);
+
+        auto mean = std::make_shared<ngraph::opset6::ReduceMean>(data, axes, true);
+        auto mean_normalization = std::make_shared<ngraph::opset6::Subtract>(data, mean);
+
+        if (!mvn_node->get_normalize_variance()) {
+            mean_normalization->set_friendly_name(m.get_match_root()->get_friendly_name());
```

Comment:
I think you should create variable match_root and use it instead of calling m.get_match_root().